test/sequential/test-timers-set-interval-excludes-callback-duration.js

```diff
@@ -9,9 +9,16 @@ const t = setInterval(() => {
   cntr++;
   if (cntr === 1) {
     common.busyLoop(100);
+    // ensure that the event loop passes before the second interval
+    setImmediate(() => assert.strictEqual(cntr, 1));
```

Comment:
Should these new callbacks also have `common.mustCall()`?

Reply:
I opted not to include since `setImmediate` is guaranteed to run and if it doesn't, others tests will fail.

Reply:
Mmm yes but I'm not sure that's guaranteed to be true for all APIs so we do need to be careful about that path of doing things,